docs/man/xrdp-chansrv.8.in

```diff
@@ -3,7 +3,7 @@
 \fBxrdp\-chansrv\fR \- \fBxrdp\fR channel server
 
 .SH "SYNTAX"
-.B xrdp\-chansrv
+.B xrdp\-chansrv [  --config /path/to/sesman.ini ]
```

Comment:
If not specified, what will happen? Refer `XRDP_CFG_PATH/sesman.ini`?

Reply:
It looks so.

Reply:
I meant for the square brackets [] to indicate it's an optional value.

Regarding your previous comment there are other possibilities:-
1. we can use environment variables to pass the chansrv configuration from sesman.
2. We can pass the sesman.ini path to chansrv via environment variable
3. We could set up a separate chansrv config file.

I'm not keen on the first option. It complicates adding variables and makes a chansrv restart complicated. I'm happy with either of the other two if you'd prefer. Or is there another option?